src/components/Popups/PopupReply.tsx

```diff
@@ -18,6 +18,25 @@ export type Props = {
     value?: string;
 };
 
+export const options = {
+    modifiers: [
+        {
+            name: 'flip',
+            options: {
+                altAxis: false,
+                fallbackPlacements: ['bottom', 'top', 'left', 'right'],
+                padding: 5,
+            },
+        },
+        {
+            name: 'offset',
+            options: {
+                offset: [0, 15],
+            },
```

Comment:
I think the offset should be with arrow.

Reply:
Should also move the arrow in this file too? Cause in the @mentions popup, the arrow is not needed.

Reply:
Sure.